kumir/examples/1-empty.kum

```diff
@@ -0,0 +1,13 @@
+﻿| Программа к учебнику информатики для 10 класса
```

Comment:
Could you convert these to UTF-8 *without BOM*? The testing fails. (Or, you could modify the grammar for BOM.)